Starter.Api/Controllers/SnakeController.cs

```diff
@@ -1,76 +1,72 @@
-        /// Every game has a unique ID that can be used to allocate resources or data you may need.
-        /// Your response to this request will be ignored.
-        /// </summary>
-        [HttpPost("start")]
-        public IActionResult Start(GameStatusRequest gameStatusRequest)
-        {
-            return Ok();
-        }
+    /// <summary>
+    /// Your Battlesnake will receive this request when it has been entered into a new game.
+    /// Every game has a unique ID that can be used to allocate resources or data you may need.
+    /// Your response to this request will be ignored.
+    /// </summary>
+    [HttpPost("start")]
+    public IActionResult Start(GameStatusRequest gameStatusRequest)
+    {
+        return Ok();
+    }
 
 
-        /// <summary>
-        /// This request will be sent for every turn of the game.
-        /// Use the information provided to determine how your
-        /// Battlesnake will move on that turn, either up, down, left, or right.
-        /// </summary>
-        [HttpPost("move")]
-        public IActionResult Move(GameStatusRequest gameStatusRequest)
-        {
-            var direction = new List<string> {"down", "left", "right", "up"};
-            var rng = new Random();
+    /// <summary>
+    /// This request will be sent for every turn of the game.
+    /// Use the information provided to determine how your
+    /// Battlesnake will move on that turn, either up, down, left, or right.
+    /// </summary>
+    [HttpPost("move")]
+    public IActionResult Move(GameStatusRequest gameStatusRequest)
+    {
+        var direction = new List<string> { "down", "left", "right", "up" };
+        var rng = new Random();
 
-            var response = new MoveResponse
-            {
-                Move = direction[rng.Next(direction.Count)],
-                Shout = "I am moving!"
-            };
-            return Ok(response);
-        }
+        var response = new MoveResponse
+        {
+            Move = direction[rng.Next(direction.Count)],
+            Shout = "I am moving!"
+        };
+        return Ok(response);
+    }
 
 
-        /// <summary>
-        /// Your Battlesnake will receive this request whenever a game it was playing has ended.
-        /// Use it to learn how your Battlesnake won or lost and deallocated any server-side resources.
-        /// Your response to this request will be ignored.
-        /// </summary>
-        [HttpPost("end")]
-        public IActionResult End(GameStatusRequest gameStatusRequest)
-        {
-            return Ok();
-        }
+    /// <summary>
+    /// Your Battlesnake will receive this request whenever a game it was playing has ended.
+    /// Use it to learn how your Battlesnake won or lost and deallocated any server-side resources.
+    /// Your response to this request will be ignored.
+    /// </summary>
+    [HttpPost("end")]
+    public IActionResult End(GameStatusRequest gameStatusRequest)
+    {
+        return Ok();
```

Comment:
We can get rid of Controllers and use `MapPost` and equivalents in the `Program.cs`